### PR TITLE
add api for tags

### PR DESCRIPTION
**Pull request by yeyn19**
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Review by leoleoasd — APPROVED** [review]
LGTM! @suntt2019 could you review this?

File: app/controller/problem.go [hunk 1]
```diff
@@ -222,6 +232,18 @@ func CreateProblem(c echo.Context) error {
 		utils.MustPutObject(file, c.Request().Context(), "problems", fmt.Sprintf("%d/attachment", problem.ID))
 	}
 
+	//base.DB.Delete(&problem.Tags)
```
Comment (suntt2019) [inline app/controller/problem.go:235]: Commented code here

File: app/controller/problem.go [hunk 2]
```diff
@@ -280,6 +302,21 @@ func UpdateProblem(c echo.Context) error {
 	problem.LanguageAllowed = strings.Split(req.LanguageAllowed, ",")
 	problem.BuildArg = req.BuildArg
 	problem.CompareScriptName = req.CompareScriptName
+
+	//base.DB.Delete(&problem.Tags)
```
Comment (suntt2019) [inline app/controller/problem.go:306]: Commented code here

**Review by suntt2019 — APPROVED** [review]
LGTM! Thanks for your contribution, and sorry for the delay.

**Pull request merged**